javaslang/src/test/java/javaslang/collection/euler/Euler67Test.java

```diff
@@ -58,7 +59,7 @@ private static int solve(String fileName) {
         return Vector.ofAll(
                 Utils.readLines(Utils.file(fileName)).map(line ->
                         Arrays.stream(line.split("\\s")).map(Integer::parseInt)
-                ).map(s -> Vector.ofAll(s::iterator))
+                ).map(s -> List.ofAll(s::iterator).toVector())
```

Comment:
I have reverted the test, but I needed to wrap the general `Iterable` to a `List`, otherwise I might not be able to go over it multiple times (e.g. it might be a `Stream`, that won't return the same `iterator` multiple times)